pdks/Gf180/gf180_hdl21/pdk_data.py

```diff
@@ -62,34 +62,50 @@ class MosParams:
     w = h.Param(dtype=h.Scalar, desc="Width in PDK Units (µm)", default=1 * µ)
     l = h.Param(dtype=h.Scalar, desc="Length in PDK Units (µm)", default=1 * µ)
     nf = h.Param(dtype=h.Scalar, desc="Number of Fingers", default=1)
-    # This unfortunate naming is to prevent conflicts with base python.
-    As = h.Param(
+
+    #! CAUTION: The subsequent parameters are not recommended for design use.
+
+    ad = h.Param(
         dtype=h.Scalar,
-        desc="Source Area",
-        default=h.Literal("int((nf+2)/2) * w/nf * 0.18u"),
+        desc="Drain Area",
+        # `0.18u` is diffusion length (m)
+        # As `nf` increments, drain region quantity is alternatively -1 or == source region quantity
+        default=h.Literal("int((nf+1)/2) * w/nf * 0.18u"),
```

Comment:
I haven't used GF180, but I assume the source area should 'increment' in size before the drain, and so this expression should lag behind the one for the drain. For example, if we test a couple number of fingers with
`nf = np.array([1, 2, 3, 4, 5, 6])`, we see these expressions return the different diffusion region quantities..

For the drain:
```
>>> np.int_((nf + 1) / 2)
array([1, 1, 2, 2, 3, 3])
```

And for the source:
```
>>> np.int_((nf + 2) / 2)
array([1, 2, 2, 3, 3, 4])
```

Reply:
I checked the definitions used by xschem, this is correct!

Reply:
Great thanks!